src/Examples/Flowsheets/HDA_flowsheet_with_distillation_solution_testing.ipynb

```diff
@@ -0,0 +1,1773 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "What sequence did the SD tool determine to solve this flowsheet with the least number of tears? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for o in order:\n",
+    "    print(o[0].name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The SequentialDecomposition tool has determined that the tear stream is the mixer outlet (s03 in the Figure above). We will need to provide a reasonable guess for this."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tear_guesses = {\n",
+    "        \"flow_mol_phase_comp\": {\n",
+    "                (0, \"Vap\", \"benzene\"): 1e-5,\n",
+    "                (0, \"Vap\", \"toluene\"): 1e-5,\n",
+    "                (0, \"Vap\", \"hydrogen\"): 0.30,\n",
+    "                (0, \"Vap\", \"methane\"): 0.02,\n",
+    "                (0, \"Liq\", \"benzene\"): 1e-5,\n",
+    "                (0, \"Liq\", \"toluene\"): 0.30,\n",
+    "                (0, \"Liq\", \"hydrogen\"): 1e-5,\n",
+    "                (0, \"Liq\", \"methane\"): 1e-5},\n",
+    "        \"temperature\": {0: 303},\n",
+    "        \"pressure\": {0: 350000}}\n",
+    "\n",
+    "# Pass the tear_guess to the SD tool\n",
+    "seq.set_guesses_for(m.fs.H101.inlet, tear_guesses)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we need to tell the tool how to initialize a particular unit. We will be writing a python function which takes in a \"unit\" and calls the initialize method on that unit."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def function(unit):\n",
+    "        unit.initialize(outlvl=idaeslog.INFO)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now ready to initialize our flowsheet in a sequential mode. Note that we specifically set the iteration limit to be 5 as we are trying to use this tool only to get a good set of initial values such that IPOPT can then take over and solve this flowsheet for us. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "seq.run(m, function)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Inline Exercise:</b>\n",
+    "We have now initialized the flowsheet. Let us run the flowsheet in a simulation mode to look at the results. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the solver object\n",
+    "solver = SolverFactory('ipopt')\n",
```

Comment:
1. As before, should we change to using `get_solver`?
2. 5000 iterations is way too many - 100 is more than enough and I would even suggest 50.

Reply:
Switched to get_solver